README.md

```diff
@@ -30,8 +30,12 @@ The **Workspace** app is a fullstack project that allows coworking space owners
 
 This repository contains the frontend part of the project. The backend part can be found [here](https://github.com/badger-99/workspace-reservation-back-end)
 
+The project Kanban board can be found [here](https://github.com/users/badger-99/projects/7/views/1)
+
```

Comment:
You could add the initial state link and the statement about the number of students:

The initial state of Kanban board can be found [here](https://github.com/badger-99/workspace-reservation-back-end/issues/12)

Group consists of 4 students:
1 - [Alfred Makongoro](https://github.com/badger-99)
2 - [Javier Hernandez](https://github.com/wickathou)
3 - [Abdul Wahab Hussain](https://github.com/Abdulwhbhussain)
4 - [Zilola Nazarova](https://github.com/Zilola-Nazarova)